projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1577,4 +1577,13 @@ public function get_is_commercial_reasons() {
 	public function get_wpcom_admin_interface() {
 		return (string) get_option( 'wpcom_admin_interface' );
 	}
+
+	/**
+	 * Returns whether the site is part of the classic view early release.
+	 *
+	 * @return string
+	 **/
+	public function get_wpcom_classic_early_release() {
+		return (string) get_option( 'wpcom_classic_early_release' );
```

Comment:
I wasn't expecting a `string` but works.

Reply:
Yeah, this does not really matter actually, as we're only check for its existence 🙈

Reply:
I decided to return boolean to better reflect the intention. Thanks!